plugin-lenovo/lphandler/update.go

```diff
@@ -72,7 +72,7 @@ func SimpleUpdate(ctx iris.Context) {
 		operationApplyTime = requestBody["@Redfish.OperationApplyTime"].(string)
 	}
 	if operationApplyTime == "OnStartUpdateRequest" && urlList[1] == "SimpleUpdate" {
-		body := `{"error":{"code": Base.1.4.Success,"message": "See @Message.ExtendedInfo for more information.","@Message.ExtendedInfo":[{"MessageId": "Base.1.4.Success"}]}}`
+		body := `{"error":{"code": Base.1.11.Success,"message": "See @Message.ExtendedInfo for more information.","@Message.ExtendedInfo":[{"MessageId": "Base.1.11.Success"}]}}`
```

Comment:
Cross check if this messageId is correct.. everywhere message id is Base.1.11.0.XXX however here it is only Base.1.11.XXX

Reply:
Made the required changes in file update.go. I can see in docs mostly Base.1.11.XXX is used instead of Base.1.11.0.XXX.